sofl/utils/run_executable.py

```
@@ -22,22 +22,47 @@
+    import shlex
+
+    executable_path = normalize_executable_path(executable)
+
+    if not executable_path:
+        logging.error("Invalid executable path: %s", executable)
+        return
+
+    # Если executable - это строка с аргументами, безопасно распарсим её
+    if isinstance(executable, str) and " " in str(executable):
+        try:
+            # Пробуем распарсить как команду с аргументами
```

Ruff (0.12.2)

Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)